charts/datadog/README.md

```diff
@@ -42,7 +42,7 @@ By default, the Datadog Agent runs in a DaemonSet. It can alternatively run insi
 To install the chart with the release name `<RELEASE_NAME>`, retrieve your Datadog API key from your [Agent Installation Instructions](https://app.datadoghq.com/account/settings#agent/kubernetes) and run:
 
 ```bash
-helm install --name <RELEASE_NAME> \
+helm install <RELEASE_NAME> \
```

Comment:
it depends if `helm2` or `helm3` is used, cf our public doc: https://docs.datadoghq.com/agent/kubernetes/?tab=helm

Reply:
@leeavital 
could you update your PR according to my comment?
thanks

Reply:
Sorry about that, I completely lost track of this PR. I've addressed the comments.